src/govuk/components/accordion/accordion.test.js

```diff
@@ -118,13 +116,46 @@ describe('/components/accordion', () => {
         expect(expandedState).toEqual(expandedStateAfterRefresh)
       })
 
-      describe('"+/-" icon', () => {
-        it('should display "+/-" in the section headings', async () => {
+      describe('"↓/↑" icon', () => {
+        it('should display "↓/↑" in the section headings', async () => {
+          await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
+
+          const numberOfExampleSections = 2
+          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-accordion-nav__chevron').length)
+
+          expect(numberOfIcons).toEqual(numberOfExampleSections)
+        })
+      })
+
+      describe('hidden comma', () => {
+        it('should contain hidden comma " ," for assistive technology', async () => {
+          await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
+
+          const numberOfExampleSections = 2
+          const numberOfCommas = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-visually-hidden:first-child').length)
+
+          expect(numberOfCommas).toEqual(numberOfExampleSections)
+        })
+      })
+
+      it('should change the (combined) Show[ this section] text to Hide[ this section ] when sections are opened', async () => {
```

Comment:
This test should not pass, but is passing because the accordion is still open from another test scenario.

We should possibly be scoping the tests differently in general, @lfdebrux will investigate this and will raise a PR against main branch to fix.

Reply:
As a backup to that action - updated the test and added a reset as well to tackle this